src/warrior.cpp

```diff
@@ -105,6 +106,10 @@ bool warrior_ai_action(CharData *ch, CharData *victim) {
             do_hitall(ch, "", 0, SCMD_HITALL);
             return true;
         }
+        if (roll > 75 && GET_SKILL(ch, SKILL_ROUNDHOUSE) && FIGHTING(ch)) {
+            do_roundhouse(ch, "", 0, 0);
```

Comment:
Might want to make this as `return do_roundhouse(...);` and have do_roundhouse return if it's successful or not.  Depending on what returning true/false means.